5_databases/week2/task1_2.sql

```diff
@@ -0,0 +1,74 @@
+USE hyf_lesson1;
+
+--1. Add a task  with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id)
+VALUES ('Be good', 'Stop beating your husband', '2023-11-16 12:00:00', '2023-11-16 12:00:00', '2023-12-01 12:00:00', 1, 3);
+
+--2. Change the title of a task
+UPDATE task
+set title = "Sleep"
+where title = "Wash clothes";
```

Comment:
Correct, but better to do with a primary key (id) so you dont edit more than one

Reply:
Changed as advised!